CONTRIBUTING.md

```diff
@@ -0,0 +1,54 @@
+# Contributing Guidelines
+
+The KUDO Operators project accepts contributions via GitHub pull requests. This document outlines the process to help get your contribution accepted.
+
+## Sign the CLA
+
+Kubernetes projects require that you sign a Contributor License Agreement (CLA) before we can accept your pull requests.
+
+Please see https://git.k8s.io/community/CLA.md for more info.
+
+## How to Contribute to an Existing Operator
+
+1. Submit an issue describing your proposed change to the operator in question.
```

Comment:
markdown nit: I think you can do 1. 1. 1. instead of 1. 2. 3. to make the future edits easier

Reply:
neat!